pg_search/src/postgres/storage/utils.rs

```diff
@@ -25,7 +25,7 @@ use std::sync::Arc;
 
 pub trait BM25Page {
     unsafe fn read_item<T: From<PgItem>>(&self, offsetno: pg_sys::OffsetNumber) -> Option<T>;
-
+    unsafe fn read_pg_item(&self, offsetno: pg_sys::OffsetNumber) -> Option<PgItem>;
```

Comment:
This isn't necessary.  There's `Page::read_item<T>()` that already exists and should be used